### ISSUE TITLE
Simplemobs Seek Lost Targets: Feedback

### ISSUE THREAD

**Issue by Atermonera** [PolarisSS13/Polaris#8092]
This is a central location for logging people's feedback on the recent changes to simplemob pathing (#8052) that allows them to traverse multiz and chase targets through doorways. This exists so that people can more easily complain if I made mobs too smart, or whine that they still don't do anything, or otherwise indicate the efficacy of the changes, and so that devs can at some point in the future look at this and say "Hey, so does this actually need any updates?"

**Comment by ZirconiumWit** [comment]
Unsure if this is a new issue or related to this, so I'll try here first - during the map test the other night, I witnessed some issues with simplemobs targeting 'nothing' or random terrain, pictured here -
![drone issues](https://user-images.githubusercontent.com/59771903/128800624-72cfc347-dfbd-4b78-baab-058ad0b98ec8.png)
![alien targeting issues](https://user-images.githubusercontent.com/59771903/128800626-59e95524-9b8e-4d5f-9610-f9151211a61f.png)


Sorry, I would have reported this sooner, but I only just found my way to your github.

**Comment by Cerebulon** [comment]
Yeah there's a major issue where the AI continues to target mobs after death if the death deletes the actual 'mob' corpse to replace it with an item such as mobs which explode/disintegrate, and 'human' mobs where the 'mob' is deleted and replaced with a dressed up dummy corpse for looting purposes.

It also appears to cause runtimes in the mob targeting system where it's attempting to call checks on a null mob.

Seems like it just needs an extra qdel check or something?

**Comment by Sypsoti** [comment]
Endless targeting of deleted mobs should be fixed with #8521
